script/Ciel/Day 5/05_01_CIEL05_4D.txt

```diff
@@ -949,7 +948,7 @@ But, are you going to be okay, young master?
 [sha:98eea8701369f9b1aa74aac35c93f922bd8b0cd7]{
 -- Page 853, Offset 13145.
 -- 「魔術だの秘蹟だのに頼らない時点でテメェらよりマシだろうよ。よしんば見つかっても<真|ア><祖|レ>とは顔見知りだ。殺される事はねぇ」
-"I'd be better off than your dumb ass even if I didn't make use of magecraft, sacraments, and all that lot.
+"I'm better off than your dumb ass even without relying on magecraft, sacraments, and all that lot.
```

Comment:
Didn't communicate this properly enough in the group PR (was really running out of time there).
"Given I don't rely on magecraft, sacraments, or any of that shit, I'm way better off than your dumb ass..."